cbmc/proofs/IotMqtt_Wait/Makefile

```diff
@@ -0,0 +1,41 @@
+ENTRY=IotMqtt_Wait
+
+ABSTRACTIONS += --remove-function-body _IotMqtt_ProcessIncomingPublish
+ABSTRACTIONS += --remove-function-body _matchEndWildcards
+ABSTRACTIONS += --remove-function-body _matchWildcards
+ABSTRACTIONS += --remove-function-body _mqttOperation_match
+ABSTRACTIONS += --remove-function-body _mqttOperation_tryDestroy
+ABSTRACTIONS += --remove-function-body _packetMatch
+ABSTRACTIONS += --remove-function-body _topicFilterMatch
+ABSTRACTIONS += --remove-function-body _topicMatch
+ABSTRACTIONS += --remove-function-body IotListDouble_Remove
+ABSTRACTIONS += --remove-function-body 'IotListDouble_Remove$$link3'
+ABSTRACTIONS += --remove-function-body IotLog_Generic
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceClose
+
+OBJS += $(ENTRY)_harness.goto
+OBJS += $(MQTT)/cbmc/proofs/mqtt_state.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_api.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_operation.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_serialize.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_subscription.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_validate.goto
+
+# One more than actual number of subscriptions
+SUBSCRIPTION_COUNT_MAX=4
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+OPERATION_COUNT_MAX=4
```

Comment:
Add blank line the comment "One more than actual number of operations in an operation list"